src/corelibs/U2Formats/src/mysql_dbi/MysqlMsaDbi.cpp

```diff
@@ -423,7 +413,6 @@ void MysqlMsaDbi::removeRows(const U2DataId &msaId, const QList<qint64> &rowIds,
 
 void MysqlMsaDbi::removeRow(const U2DataId &msaId, qint64 rowId, U2OpStatus &os) {
     MysqlTransaction t(db, os);
-    Q_UNUSED(t);
```

Comment:
Why'd you removed it? It's really unused

Reply:
It is not unused because constructor is a function call with side effects. I think the problem was some day ago with old compilers. 
Today if we have a really unused variable the build (on Linux) will fail.